rechnung/settings.py

```diff
@@ -45,6 +45,9 @@
     "invoices_dir": "invoices",
     "logo_asset_file": "logo.svg",
     "policy_attachment_asset_file": "policy.pdf",
+    "bank_statements_dir": "bank_statements",
+    "payments_dir": "payments",
+    "decimal_quantization": ".01",
```

Comment:
Isn't his obvious and don't need it's own option?

Reply:
But it should be created on init and also should be configurable. I.e. if your bank_statements are stored somewhere else in the system and not necessarily in the applications data folder.